src/util/bip32.rs

```diff
@@ -187,6 +202,22 @@ impl serde::Serialize for ChildNumber {
     }
 }
 
+/// Parse a BIP32 derivation path.
+pub fn parse_derivation_path(path: &str) -> Result<Vec<ChildNumber>, Error> {
+    let mut parts = path.split("/");
+    // First parts must be `m`.
+    if parts.next().unwrap() != "m" {
```

Comment:
This would panic on an empty string?

Reply:
Don't think so. Empty string splits into single element that's an empty string.

https://doc.rust-lang.org/std/primitive.str.html#examples-26
Second example.